pymdownx/superfences.py

```diff
@@ -679,6 +677,7 @@ def search_nested(self, lines):
                 if m is not None:
 
                     # Parse options
+                    self.lang = m.group('lang') or ''
```

Comment:
I understand this was moved out to be in one place, but I would prefer that the language handling is parsed in the `handle_attrs` and `parse_options` functions.